src/redux/slice/sliceCart.js

```diff
@@ -27,24 +27,24 @@ const orderSlice = createSlice({
   initialState,
   reducers: {
     increaseProduct(state, action) {
-      const itemIndex = state.cartItems.findIndex(cartElement => cartElement.id === action.payload.id);
+      const itemIndex = state.cartItems.findIndex(cartElement => cartElement.id === action.payload.id && cartElement.color === action.payload.color && cartElement.size === action.payload.size);
```

Comment:
The condition to check if item type is already in the cart is quite complex. We can make it easier to read understand by using a named function to check the condition:
```
const isItemTypeInCart = (cartItems, newItem) => {
  return cartItems.findIndex(cartElement => cartElement.id === newItem.id && cartElement.color === newItem.color && cartElement.size === newItem.size) !== -1;
}

increaseProduct(state, action) {
   const isItemAlreadyInCart = isItemTypeInCart(state.cartItems, action.payload);
   if (isItemAlreadyInCart) {
          state.cartItems[itemIndex].cartQuantity += 1;
        } else {
         ...
```